web_programming/fetch_quotes.py

```diff
@@ -0,0 +1,33 @@
+"""
+This file fetches quotes from the " ZenQuotes API ".
+It does not require any API key as it uses free tier.
+
+For more details and premium features visit:
+    https://zenquotes.io/
+"""
+
+import requests
+import pprint
```

Comment:
```suggestion
import pprint

import requests
```

Reply:
Thank you for the edits, but what was the reason and how can I avoid this in future.

Reply:
https://pycqa.github.io/isort